### PR TITLE
feat: Add examples

### PR DESCRIPTION
**Pull request by internetova**
## Changes
- Add examples.
- Features such as Suggestions, Search, ReverseSearch, Bicycle, Driving have been removed from it.

## Video

https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/39735343/8dc5a6ab-38a5-42e5-baf6-dc12222635be

## Checklist for self-check
- [x] Commits and PRs have been filed according to [the rules on the project](https://github.com/surfstudio/surf-flutter-app-template#workflow-in-a-repository).
- [x] The author is marked as an assigne and assigned mandatory reviewers.
- [x] Required labels marked
- [x] Specified related tasks and/or related PRs.
- [x] Specified Changes.
- [x] Attached videos/screenshots demonstrating the fix/feature.
- [x] All unspecified fields in the PR description deleted.
- [ ] New code covered by tests.

## Checklist for reviewers
- [ ] CI passed successfully _(with a green check mark)_.
- [ ] PR is atomic, by volume no more than 400 (+-) corrected lines (not including codogen).

Design:
- [ ] System design corresponds to the agreements on structure and architecture on the project.
- [ ] The code is decomposed into necessary and sufficient components.

Functionality:
- [ ] The code solves the problem.
- [ ] Any changes to the user interface are reasonable and look good.

Complexity:
- [ ] The code is clear, easy to read, functions are small, no more than 50 lines.
- [ ] The logic is not overcomplicated, there is no overengineering (no code sections that may be needed in the future, but no one knows about it).

Tests:
- [ ] Updated or added tests for mandatory components.
- [ ] The tests are correct, helpful, and well designed/developed.

Naming:
- [ ] The naming of variables, methods, classes and other components is understandable.

Comments:
- [ ] The comments are understandable and helpful.

Documentation:
- [ ] All labels are correct
- [ ] Technical documentation updated (after approval, updates last reviewer).

### CODE REVIEW

File: example/lib/examples/clusterized_placemark_collection_page.dart [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+                        if (!mapObjects.any((el) => el.mapId == mapObjectId)) {
+                          return;
+                        }
+
+                        final mapObject = mapObjects
+                          .firstWhere((el) => el.mapId == mapObjectId) as ClusterizedPlacemarkCollection;
+
+                        setState(() {
+                          mapObjects[mapObjects.indexOf(mapObject)] = mapObject.copyWith(placemarks: [
+                            PlacemarkMapObject(
```
Comment (Sadhorsephile) [inline example/lib/examples/clusterized_placemark_collection_page.dart:163]: can we replace it with some more readable generation? 
Smth like this:
```dart
 final pos = <({String name, double lat, double lon})>[
    (name: 'map1', lat: 30.1, lon: 30.2),
    (name: 'map2',lat: 32.1, lon: 30.2),
    (name: 'map3',lat: 33.1, lon: 29.2),
  ];
 
  mapObjects[mapObjects.indexOf(mapObject)] = 
  mapObject.copyWith(placemarks: pos.map((e) =>
PlacemarkMapObject(
                              mapId: MapObjectId(e.name),
                              point: Point(latitude: e.lat, longitude: e.lon),
                              icon: PlacemarkIcon.single(PlacemarkIconStyle(
                                image: BitmapDescriptor.fromAssetImage('lib/assets/place.png'),
                                scale: 1
                              ))
                            )

File: example/lib/examples/map_controls_page.dart [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+  static const Point _point = Point(latitude: 59.945933, longitude: 30.320045);
+  final animation = const MapAnimation(type: MapAnimationType.smooth, duration: 2.0);
+
+  bool tiltGesturesEnabled = true;
+  bool zoomGesturesEnabled = true;
+  bool rotateGesturesEnabled = true;
+  bool scrollGesturesEnabled = true;
+  bool nightModeEnabled = false;
+  bool fastTapEnabled = false;
+  bool mode2DEnabled = false;
```
Comment (Sadhorsephile) [inline example/lib/examples/map_controls_page.dart:35]: var

File: example/lib/examples/map_controls_page.dart [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+          "all": ["landscape"]
+        },
+        "stylers": {
+          "color": "f00",
+          "saturation": 0.5,
+          "lightness": 0.5
+        }
+      }
+    ]
+  ''';
```
Comment (Sadhorsephile) [inline example/lib/examples/map_controls_page.dart:57]: I think here should be the link to docs, where can be founded all possible values of this config

File: example/lib/examples/map_controls_page.dart [hunk 4]
```diff
@@ -0,0 +1,450 @@
...
+    if (old.horizontal == HorizontalAlignment.right && old.vertical == VerticalAlignment.bottom) {
+      return const MapAlignment(horizontal: HorizontalAlignment.right, vertical: VerticalAlignment.top);
+    }
+
+    if (old.horizontal == HorizontalAlignment.right && old.vertical == VerticalAlignment.top) {
+      return const MapAlignment(horizontal: HorizontalAlignment.center, vertical: VerticalAlignment.top);
+    }
+
+    if (old.horizontal == HorizontalAlignment.center && old.vertical == VerticalAlignment.top) {
+      return const MapAlignment(horizontal: HorizontalAlignment.left, vertical: VerticalAlignment.top);
```
Comment (Sadhorsephile) [inline example/lib/examples/map_controls_page.dart:98]: Maybe it'll look better with switch

File: example/lib/examples/map_controls_page.dart [hunk 5]
```diff
@@ -0,0 +1,450 @@
...
+              controller = yandexMapController;
+
+              await controller.setMinZoom(zoom: 2);
+              await controller.setMaxZoom(zoom: 18);
+
+              final cameraPosition = await controller.getCameraPosition();
+              final minZoom = await controller.getMinZoom();
+              final maxZoom = await controller.getMaxZoom();
+
+              print('Camera position: $cameraPosition');
```
Comment (Sadhorsephile) [inline example/lib/examples/map_controls_page.dart:138]: You use `debugPrint` in other places

File: example/lib/examples/polygon_map_object_page.dart [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+                    ControlButton(
+                      onPressed: () async {
+                        if (mapObjects.any((el) => el.mapId == mapObjectId)) {
+                          return;
+                        }
+
+                        final mapObject = PolygonMapObject(
+                          mapId: mapObjectId,
+                          polygon: const Polygon(
+                            outerRing: LinearRing(points: [
```
Comment (Sadhorsephile) [inline example/lib/examples/polygon_map_object_page.dart:56]: Can we move points mock list to separate variable?

File: example/.fvm/flutter_sdk
Comment (maxim-sysoev) [inline example/.fvm/flutter_sdk:1]: must be in git ignore

**Comment by internetova** [comment]
https://github.com/surfstudio/yandex-mapkit-lite-flutter/pull/3

**Pull request closed without merging**